src/app/pages/details/common/get-this-title-files/order-print-copy/order-print-copy.js

```diff
@@ -1,40 +1,35 @@
 import React from 'react';
 import {RawHTML} from '~/components/jsx-helpers/jsx-helpers.jsx';
 import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import useAmazonAssociatesLink from './amazon-associates-link';
 import './order-print-copy.css';
 
-function Header({icon, heading, content}) {
+function Header({entry, disclosure}) {
+    const icon = entry.buttonUrl ? 'users' : 'user';
+
     return (
         <React.Fragment>
             <h1>
                 <FontAwesomeIcon icon={icon} />
-                {heading}
+                {entry.heading}
             </h1>
-            <RawHTML html={content} />
+            <RawHTML html={disclosure || entry.content} />
         </React.Fragment>
     );
 }
 
-function Button({href, text, buttonClass, onClick}) {
-    return (
-        <a className={`btn ${buttonClass}`} href={href} onClick={onClick} data-track="Print">
-            {text}
-        </a>
-    );
-}
+function PhoneBox({entry, amazonDataLink, amazonLink, closeAfterDelay}) {
+    const [href, disclosure] = entry.buttonUrl ?
+        [entry.buttonUrl, ''] :
+        [amazonDataLink.url || amazonLink, amazonDataLink.disclosure];
 
-function PhoneBox({entry, amazonLink, closeAfterDelay}) {
     return (
         <a
             className="box"
-            href={entry.buttonUrl || amazonLink}
```

Comment:
@RoyEJohnson is it safe to remove the amazon_link from the CMS (from os-webview perspective)?

Reply:
I have removed all references to it from my code now.